scripts/linters/general_purpose_linter.py

```diff
@@ -123,18 +123,6 @@
         'excluded_files': (),
         'excluded_dirs': ()
     },
-    {
-        'regexp': re.compile(r'\b(browser.sleep)\('),
```

Comment:
Removed this check because it also disallows browser.sleep() calls

Reply:
Remove it's tests as well from general_purpose_linter_test.py